app/services/katello/pulp3/repository.rb

```diff
@@ -135,8 +135,8 @@ def delete(href = repository_reference.try(:repository_href))
       end
 
       def sync
-        repository_sync_url_data = api.class.client_module::RepositorySyncURL.new(repository: repository_reference.repository_href, mirror: repo.root.mirror_on_sync)
-        [api.remotes_api.sync(repo.remote_href, repository_sync_url_data)]
+        repository_sync_url_data = api.class.client_module::RepositorySyncURL.new(remote: repo.remote_href, mirror: repo.root.mirror_on_sync)
+        [api.repositories_api.sync(repository_reference.repository_href, repository_sync_url_data)]
```

Comment:
am i reading this right, the specification of remote and repository_href's are swapped?

Reply:
@jlsherrill there are now, yeah